wasmer2/wasmer2Executor.go

```diff
@@ -112,8 +106,29 @@ func (wasmerExecutor *Wasmer2Executor) NewInstanceFromCompiledCodeWithOptions(
 	compiledCode []byte,
 	options executor.CompilationOptions,
 ) (executor.Instance, error) {
-	// return NewInstanceFromCompiledCodeWithOptions(compiledCode, options)
-	return nil, errors.New("NewInstanceFromCompiledCodeWithOptions not implemented")
+	var c_instance *cWasmerInstanceT
+
+	if len(compiledCode) == 0 {
+		var emptyInstance = &Wasmer2Instance{cgoInstance: nil}
+		return emptyInstance, newWrappedError(ErrInvalidBytecode)
```

Comment:
Return `nil` here instead of `emptyInstance`.

Reply:
done